src/systems/userinput/devices/app-aware-touchscreen.js

```diff
@@ -176,8 +176,13 @@ export class AppAwareTouchscreenDevice {
         );
         break;
       case MOVE_CAMERA_JOB:
-        assignment.delta[0] += touch.clientX - assignment.clientX;
-        assignment.delta[1] += touch.clientY - assignment.clientY;
+	if (window.APP.store.state.preferences.invertTouchscreenCameraMove) {
```

Comment:
I'm going to move this check closer to where the camera rotation is applied so so that other places where these `delta` values are read still get the non-inverted values.

Reply:
Awesome, great news! Thanks!